gapic/api/artman_logging.yaml

```diff
@@ -9,8 +9,10 @@ common:
   output_dir: ${REPOROOT}/artman/output
   gapic_api_yaml:
     - ${REPOROOT}/googleapis/google/logging/v2/logging_gapic.yaml
+  enable_batch_generation: True
 java:
   final_repo_dir: ${REPOROOT}/gcloud-java/gcloud-java-logging
+  batch_dir_name: gcloud-java/gcloud-java-logging
```

Comment:
But that seems redundant with final_repo_dir... Maybe you could break apart the field?

* final_repo_name (maybe move this to the language yaml instead of having it for every api x language)
* repo_subdir_name

And then repo_subdir_name is used by the batch pipeline, and both are used for whatever task copies the code to the final repo.